slasher/db/min_max-span_test.go

```diff
@@ -0,0 +1,118 @@
+package db
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"testing"
+
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+)
+
+type spanMapTestStruct struct {
+	validatorID uint64
+	spanMap     *ethpb.EpochSpanMap
+}
+
+var spanTests []spanMapTestStruct
+
+func init() {
+	spanTests = []spanMapTestStruct{
```

Comment:
Maybe each of these can have better descriptions as to what exactly they are or maybe why the values were selected this way

Reply:
i don't have the detection and update phase in place. so those are just examples. will be adding detection and logic in the next pr